simpleclient_hikaricp/src/main/java/io/prometheus/client/hikaricp/HikariCPCollector.java

```diff
@@ -0,0 +1,39 @@
+package io.prometheus.client.hikaricp;
+
+import com.zaxxer.hikari.metrics.PoolStats;
+import io.prometheus.client.Collector;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+class HikariCPCollector extends Collector {
+  private final PoolStats poolStats;
+  private final List<String> labelNames;
+  private final List<String> labelValues;
+
+  HikariCPCollector(String poolName, PoolStats poolStats) {
+    this.poolStats = poolStats;
+    this.labelNames = Collections.singletonList("pool");
+    this.labelValues = Collections.singletonList(poolName);
+  }
+
+  @Override
+  public List<MetricFamilySamples> collect() {
+    return Arrays.asList(
+            createSample("hikaricp_active_connections", "Active connections", poolStats.getActiveConnections()),
+            createSample("hikaricp_idle_connections", "Idle connections", poolStats.getIdleConnections()),
+            createSample("hikaricp_pending_threads", "Pending threads", poolStats.getPendingThreads()),
+            createSample("hikaricp_total_connections", "Total connections", poolStats.getTotalConnections())
```

Comment:
I'd suggest removing the `total_` from the name then, and clarifying the docstring.